galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -50,6 +58,24 @@ class NamespaceViewSet(
 
     filterset_class = NamespaceFilter
 
+    def create(self, request, *args, **kwargs):
+        groups = []
+        for account in request.data['groups']:
+            if account == 'system:partner-engineers':
+                groups.append(account)
+            else:
+                group, _ = auth_models.Group.objects.get_or_create_identity(
+                        RH_ACCOUNT_SCOPE, account)
+                group.save()
```

Comment:
`get_or_create_identity` retrieves an existing record or creates a new one. `save()` is not needed here.